docs/dmca/index.rst

```diff
@@ -4,32 +4,36 @@ DMCA Takedown Policy
 These are the guidelines that Read the Docs follows when handling DMCA takedown
 requests and takedown counter requests. If you are a copyright holder wishing to
 submit a takedown request, or an author that has been notified of a takedown
-request, please familiarize yourself with `our process <Takedown Process_>`_.
+request, please familiarize yourself with `our process`__.
```

Comment:
Why we don't leave the extended name here? Isn't it clearer to understand where this will go?

Reply:
(the same for the other changes in this file)

Reply:
rstcheck didn't recognize those and marked it as errors. And I didn't find that syntax on the quickref http://docutils.sourceforge.net/docs/user/rst/quickref.html#internal-hyperlink-targets.

Reply:
`our process <Takedown Process_>`_

links to the "Takedown Process" title (L20 in `dcma/index.rst`)

If you remove the title from the link, like this

    `our process`__

where that link will follow you? If you build the docs locally, you will find that link doesn't work.

Reply:
My mouse is on `our process` but it doesn't appear on the screeenshot :grin: